tests/ui/perfherder/graphs-view/graphs_view_test.jsx

```diff
@@ -216,6 +221,42 @@ test('Using select query param displays tooltip for correct datapoint', async ()
   expect(platform).toHaveTextContent(testData[0].platform);
 });
 
+test("Alert's ID can be copied to clipboard from tooltip", async () => {
+  const selectedDataPoint = {
+    signature_id: testData[0].signature_id,
+    dataPointId: testData[0].data[5].id,
+  };
+
+  Object.assign(navigator, {
+    clipboard: {
+      writeText: jest.fn(),
+    },
+  });
+  const { getByTestId, queryAllByTitle } = graphsViewControls(
+    graphData,
+    false,
+    undefined,
+    undefined,
+    selectedDataPoint,
+  );
+
+  const graphContainer = await waitFor(() => getByTestId('graphContainer'));
+  expect(graphContainer).toBeInTheDocument();
+
+  const graphTooltip = await waitFor(() => getByTestId('graphTooltip'));
+  expect(graphTooltip).toBeInTheDocument();
+
+  const copyIdButtons = await waitFor(() =>
+    queryAllByTitle('Copy Alert Summary id'),
+  );
+  expect(copyIdButtons).toHaveLength(1);
+
+  fireEvent.click(copyIdButtons[0]);
+
+  const alertID = alertSummaries[0].id;
```

Comment:
Wondering if it's not possible to use **testdata** instead of **alertSummaries** so that you don't need to import it just for the following check.

Reply:
**testdata** contains only datapoint information, while **alertSummaries** contains alerts, and each alert includes the datapoint ID where the alert was recorded.
I need the alertSummaries to retrieve the alert ID.